### ISSUE TITLE
Where available, indicate whether a component is the latest release.

### ISSUE THREAD

**Issue by kenguest** [getmatlock/Matlock-Breadcrumbs#2]
Would it be possible to have matlock indicate if a component is the latest version?

It should be possible to get the latest version number from githubs's api via:
https://api.github.com/repos/organisationname/repositoryname/releases/latest

**Comment by onassar** [comment]
What do you mean by component? Do you mean a GitHub library? Def possible to compare the stated version number (if it's found) with the latest release version number. Thats a rad idea; maybe show a little badge/icon next to it when they match to indicate that it's the most recent version.

**Comment by kenguest** [comment]
yes that's exactly what I mean, thanks.  I think it'd help people to check if they need to update their libraries if too far behind current.

**Comment by onassar** [comment]
Yep thats a cool idea. Will add to the roadmap :)
